RIOT-OS/RIOT
# boards/nucleo-f303ze: add pinout

### Contribution description

This PR add missing pinout from UM1947 for nucleo-f303ze.

### Testing procedure

Generate doc and see if everything is fine.

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-f303ze.html
```

### Issues/PRs references

None
